README.md

```diff
@@ -1,85 +1,341 @@
+	[IterableAPI initializeWithApiKey:@"<your-api-key>" launchOptions:launchOptions config:config]
+	```
+	
+	
+	See the Iterable guide on how to setup your Iterable push integration and obtain push integration name [here](https://support.iterable.com/hc/en-us/articles/115000315806-Setting-Up-iOS-Push-Notifications).	
+	
+2. ##### Set userId or email. 
+
+	Once you know the email or userId of the user, set the value.
+
+	Swift:
+	
+	```swift
+	IterableAPI.email = "user@example.com"
 	```
-	IterableAPI.initialize(apiKey: "your-api-key", optionalParameters.....)
+
+	Objective-C:
+
+	```objective-c
+	IterableAPI.email = @"user@example.com";
 	```
-  * Ideally, you will call this from inside `application:didFinishLaunchingWithOptions:` and pass in `launchOptions`. This will let the SDK automatically track a push open for you if the application was launched from a remote Iterable push notification. 
 	
-2. Once you know the email *(Preferred)* or userId of the user, set the value.
+3. ##### Send Remote Notification Token to Iterable
+
+	* See [Apple Notification Guide](https://developer.apple.com/documentation/usernotifications) regarding how to register for remote notifiations.
+	* In your `AppDelegate`’s [application:didRegisterForRemoteNotificationsWithDeviceToken:](https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1622958-application) method, send the token obtained to Iterable.
+
+	Swift:
+	
+	```swift
+	func application(_ application: UIApplication, didRegisterForRemoteNotificationsWithDeviceToken deviceToken: Data) {
+        IterableAPI.register(token: deviceToken)
+    }
+    ```
+
+	Objective-C:
+	
+	```objective-c
+	- (void)application:(UIApplication *)application didRegisterForRemoteNotificationsWithDeviceToken:(NSData *)deviceToken {
+    	[IterableAPI registerToken:deviceToken];
+	}
+    ```
+    See example in sample app delegate [here](./sample-apps/swift-sample-app/swift-sample-app/AppDelegate.swift).
 
+Congratulations! You can now send remote push notifications to your device from Iterable!
+
+# Using the SDK
+
+1. ##### Handle Push Notifications
+
+	When the user taps on the push notification or one of the action buttons, the system calls `UNUserNotificationCenterDelegate`'s [userNotificationCenter:didReceiveNotificationResponse:withCompletionHandler:](https://developer.apple.com/documentation/usernotifications/unusernotificationcenterdelegate/1649501-usernotificationcenter?language=objc). Pass this call to **`IterableAppIntegration`** to track push open event and perform the associated action (see below for custom action and URL delegates).
+	
+	Swift:
+	
+	```swift
+	public func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+   		IterableAppIntegration.userNotificationCenter(center, didReceive: response, withCompletionHandler: completionHandler)
```

Comment:
Different indentation is used in all Swift and Objective-C snippets: tabs for Swift, spaces for Objective-C. Is this intentional?